src/treelearner/cost_effective_gradient_boosting.hpp

```diff
@@ -107,5 +108,5 @@ class CostEfficientGradientBoosting {
   std::vector<uint32_t> feature_used_in_data_;
 };
 
-}  // namespace LightGBM
```

Comment:
Wrong line number due to added blank line. Original cpplint issue was about `#endif` line (112 now).